fuzz/config.toml

```diff
@@ -0,0 +1,28 @@
+[core_settings]
+log_level = "error"
```

Comment:
If I am not wrong, this will be ignored right? Maybe better to remove it to not add confusion.

Reply:
true!